payments_payu/provider.py

```diff
@@ -61,7 +65,9 @@ def add_new_status(payment, new_status):
         old_extra_data["statuses"] = []
     old_extra_data["statuses"].append(new_status)
     payment.extra_data = json.dumps(old_extra_data, indent=2)
-    payment.save()
+    payment._change_reason = 'PayU Payments: add_new_status'  # For django-simple-history
+    payment.save(update_fields=["extra_data"])
+    payment._change_reason = None
```

Comment:
Also this (as well as in all the other cases) may be a bit safer:

```suggestion
    payment._change_reason, payment_change_reason_original = 'PayU Payments: add_new_status'  # For django-simple-history, payment._change_reason
    payment.save(update_fields=["extra_data"])
    payment._change_reason = payment_change_reason_original
```

But I may be overengineering this...

Reply:
Or maybe

```suggestion
    payment._change_reason += 'PayU Payments: add_new_status'  # For django-simple-history
    payment.save(update_fields=["extra_data"])
    payment._change_reason = None
```

:thinking: